assessment.sql

```diff
@@ -5,24 +5,53 @@ CREATE DATABASE "sql-crud-assessment";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname,given_name,gender,height,weight,dob,eye_color,hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
-
+-- CREATE TABLE licensees(
```

Comment:
You've commented out the part of a CREATE TABLE query where it actually says `CREATE TABLE` and specifies the table name. This also means that the closing paren to follow will be unmatched, but the interpreter will stop and complain long before that.

Uncomment this line and you're good. 3/4